### ISSUE TITLE
Update for Fable 4

### ISSUE THREAD

**Issue by OnurGumus** [elmish/debugger#51]
### Description

At the moment this won't compile with Fable 4 failing with below error
```
./src/Client/fable-output/fable_modules/Fable.Elmish.Debugger.3.3.0/debugger.fs(127,12): (127,51) error FSHARP: Type mismatch. Expecting a
    'Program<'a,'model,'msg,'view> -> Program<'a,'model,'msg,'view>'    
but given a
    '(('msg -> bool) * ('model -> unit) -> ('msg -> bool) * ('model -> unit)) -> Program<'b,'model,'msg,'c> -> Program<'b,'model,'msg,'c>'    
The type 'Program<'a,'model,'msg,'view>' does not match the type '('msg -> bool) * ('model -> unit) -> ('msg -> bool) * ('model -> unit)' (code 1)
./src/Client/fable-output/fable_modules/Fable.Elmish.Debugger.3.3.0/debugger.fs(127,42): (127,51) error FSHARP: Type mismatch. Expecting a
    '('model -> Sub<'msg>) -> 'model -> Sub<'msg>'    
but given a
    '('model -> Cmd<'msg>) -> 'model -> Cmd<'msg>'    
The type 'SubId * Subscribe<'msg>' does not match the type 'Effect<'msg>' (code 1)
```

**Comment by et1975** [comment]
It seems you're trying to use Debugger v3 with Elmish v4 - that is not supported, but I haven't got around to aligning v4 debugger with new subscriptions yet.

**Comment by et1975** [comment]
Released in 4.0.0-beta-2
